### PR TITLE
Completed Javascript-objects

### PR DESCRIPTION
**Pull request by PhilChoe**
image

### CODE REVIEW

File: prep-javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+console.log(person);
+
+const fullName = person.firstName + ' ' + person.lastName;
+console.log("The person's full name is", fullName);
+
+person.job = 'Self-employed';
+console.log("The person's job is", person.job);
+
+person['previousJob'] = 'a student';
+console.log("The person's previous job was", person['previousJob']);
```
Comment (shawnkost) [inline prep-javascript-objects/main.js:15]: Missing one final console log for the whole `person` object.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just missing one `console.log` here!

**Comment by PhilChoe** [comment]
image

**Comment by PhilChoe** [comment]
image

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**